src/forms/services/form-submission.service.ts

```
@@ -0,0 +1,1184 @@
+              'fv.fieldId = :fieldId AND (fv.dropdownValue ? :value OR fv.value = :value)',
+              {
+                fieldId,
+                value
+              }
+            );
+          }
+          break;
+
+        case FieldType.TEXT:
+          // Check textValue column first, then fall back to value column with ILIKE
+          const textSearchValue = expectedValue.toString();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)